packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -611,13 +632,21 @@ function getActiveIndex(activeIds: string[], highlightedList: any[]): number {
 
 function getActiveElement(activeIds: string[], itemList: any[]): any {
   return itemList.find(item => {
-    if (item.allAttributeIds) {
+    if ('allAttributeIds' in item) {
       return isEqual(item.allAttributeIds.sort(), activeIds.sort());
+    } else if (isEqualId(item, activeIds)) {
+      return true;
+    } else if ('attributes' in item) {
+      return item.attributes.find((attribute: any) => isEqualId(attribute, activeIds));
     }
-    return isEqual([getId(item)].sort(), activeIds.sort());
+    return false;
   });
 }
 
+function isEqualId(item: any, id: string[]): boolean {
```

Comment:
Try to avoid adding `any`. In this case, it seems like `item` should be an object with an `id` property, right?

Reply:
Also, `id` is an array, meaning multiple. Therefore, prefer `ids` for the name.

And doesn't that mean that this should be named `isEqualIds`?